tests/conftest.py

```diff
@@ -191,6 +199,22 @@ def wait_for_window_of(self, wmclass):
                 'waiting for hook triggered by client \"{}\"'.format(wmclass))
         return line[-1]
 
+    def wait_queue_empty(self, timeout=5):
+        """Wait for the hlwm process to be in sleep state"""
+        pid = self.hlwm_process.proc.pid
+        ps_cmd = 'ps --no-headers -o state -p {}'.format(pid)
```

Comment:
How platform dependent is this? Maybe consider using [pslib](https://psutil.readthedocs.io/en/latest/) instead?